source/fem-dem/parameters_cfd_dem.cc

```diff
@@ -116,6 +116,10 @@ namespace Parameters
                       "100",
                       Patterns::Integer(),
                       "dem-cfd coupling frequency");
+    prm.declare_entry("vans model",
+                      "modelB",
```

Comment:
Did modelB show higher accuracy?

Reply:
No, both show similar behavior. I just put it as default but we can change it. If we put model A as default than all vans and cfd-dem tests will fail because the output will change slightly. That is why I didn't do that.

Reply:
We already did the development initially for model B, so I think that's a correct choice to keep B as default. To be honest, we no clue yet which one might be better...